src/sentry/web/frontend/error_page_embed.py

```diff
@@ -146,14 +146,14 @@ def dispatch(self, request):
         if form.is_valid():
             # TODO(dcramer): move this to post to the internal API
             report = form.save(commit=False)
-            report.project = key.project
+            report.project_id = key.project_id
             report.event_id = event_id
 
-            event = eventstore.get_event_by_id(report.project.id, report.event_id)
+            event = eventstore.get_event_by_id(report.project_id, report.event_id)
 
             if event is not None:
-                report.environment = event.get_environment()
-                report.group = event.group
+                report.environment_id = event.get_environment().id
+                report.group_id = event.group.id
```

Comment:
At this point, the group should always be set, but for consistency I'd go with `event.group_id`.